packages/react-components/src/trace-explorer/trace-explorer-time-range-data-widget.tsx

```diff
@@ -0,0 +1,353 @@
+import * as React from 'react';
+import { TimeRangeUpdatePayload } from 'traceviewer-base/lib/signals/time-range-data-signal-payloads';
+import { signalManager, Signals } from 'traceviewer-base/lib/signals/signal-manager';
+import { Experiment } from 'tsp-typescript-client';
+import { TimeRange } from 'traceviewer-base/lib/utils/time-range';
+
+export interface ReactTimeRangeDataWidgetProps {
+    id: string,
+    title: string,
+}
+
+export interface ReactTimeRangeDataWidgetState {
+    activeExperiment?: string,
```

Comment:
the activeExperiment (which is an experiment uuid) is also part of the activeData. Duplicate information which might cause to inconsistencies. Only use activeData.